source/_integrations/smtp.markdown

```diff
@@ -113,12 +113,16 @@ burglar:
       data:
           title: "Intruder alert"
           message: "Intruder alert at apartment!!"
+          target:
+            - "my_intruder_alert@gmail.com"
```

Comment:
Please use `example.com`, as that domain is specially reserved for documentation purposes.

```suggestion
            - "my_intruder_alert@example.com"
```